src/site/content/en/fast/codelab-replace-gifs-with-video/index.md

```diff
@@ -155,11 +155,32 @@ Update the `<video>` with `<source>` elements that link to your cat-herd videos:
 
 {% Aside %}
 Browsers don't speculate about which `<source>` is optimal, so the order
-of `<source>`s matters. For example, if you specify an MP4 video first and the
+of `<source>` elements matters. For example, if you specify an MP4 video first and the
 browser supports WebM, browsers will skip the WebM `<source>` and use the MPEG-4
 instead. If you prefer a WebM `<source>` be used first, specify it first!
 {% endAside %}
 
+## Add captions and descriptions
+One last detail to take care of: accessibility.
+With GIFs, you can provide alternative text
+to describe the image to users with visual impairments.
+For videos, you need two elements:
+- A `<track kind="descriptions">` element that links to a [WebVTT](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API)
+  file describing the visual information in the video
+- A`<track kind="captions">` element that links to a WebVTT file
```

Comment:
"links to a WebVTT file indicating that there isn't any audio" This is a little unclear to me. What goes in this file?

Reply:
I was under the impression that it's best practice to show `[No audio]` for the duration of the clip to tell hard of hearing users explicitly that they're not missing anything. I can add a sample pending response to the comment above.

Reply:
sgtm